Dockerfile-postgresql

```diff
@@ -5,11 +5,13 @@
 # the on-disk data in volumes. So the base image can not currently be upgraded
 # without users needing a manual pgdump and restore.
 
+# https://hub.docker.com/r/groonga/pgroonga/tags
+FROM groonga/pgroonga:2.3.4-alpine-14-slim
```

Comment:
Hmm, the 2.3.4 here seems like we'd want it to be a variable; we don't want ot get stuck pinned on an old version.

Reply:
Ok, I made it configurable. 
Now you can specify which pgroona base image the image is built from, i.e.:

    docker build -f Dockerfile-postgresql -t varac/zulip-postgresql:13 --build-arg PGROONGA_VERSION=2.3.4-alpine-13-slim .